.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+**Story Points: 0 | 1 | 2 | 3 | 5 | 8 | 13 | 21**
+
+Risk: HIGH | MEDIUM | LOW
+
+Priority: HIGH | MEDIUM | LOW
+
+## Description
+*Description of the issue*
+
+## Mockup
+
+![Mockup image]()
+*Image describing the PR*
+
+## Acceptance Test
+
+| Test Reference | Test Date | Tester | Test Steps | Expected Result | Actual Result |
+|----------------|-----------|--------|------------|-----------------|---------------|
+| 1              |           |        |            |                 |               |
+| 2              |           |        |            |                 |               |
+| 3              |           |        |            |                 |               |
+| 4              |           |        |            |                 |               |
+| 5              |           |        |            |                 |               |
+
+# Tasks
+
+- [ ] **Unit Tests** - *x* Ideal hour(s) *TASK #*
+- [ ] **Android Tax View/GUI** - *x* Ideal hour(s) *TASK #*
```

Comment:
Maybe have generic task names in this template instead of specific examples.

Reply:
Noted, fixed